releases/release-5.3.0.md

```diff
@@ -208,7 +209,7 @@ In v5.3, the key new features or improvements are as follows:
 
 - **Save and restore the on-site information of a cluster**
 
-    When you locate and troubleshoot the issues of a TiDB cluster, you often need to provide information on the system and the query plan. To help you get the information and troubleshoot cluster issues in a more convenient and efficient way, the `PLAN REPLAY` command is introduced in TiDB v5.3.0. This command enables you to easily save and restore the on-site information of a cluster, improves the efficiency of troubleshooting, and helps you more easily archive the issues for management.
+    When you locate and troubleshoot the issues of a TiDB cluster, you often need to provide information on the system and the query plan. To help you get the information and troubleshoot cluster issues in a more convenient and efficient way, the `PLAN REPLAYER` command is introduced in TiDB v5.3.0. This command enables you to easily save and restore the on-site information of a cluster, improves the efficiency of troubleshooting, and helps you more easily archive the issues for management.
```

Comment:
Nice catch!
A similar typo exists in Line 8 of `sql-plan-replayer.md`. Can you fix that in this pull request?